package.json

```diff
@@ -33,6 +34,7 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.0",
+    "react-truncate": "^2.4.0",
```

Comment:
generelt skeptisk til å ta inn sånne små utility-pakker som ikke har vært oppdatert på 2 år. Det er ganske enkelt å lage en egen komponent hvis vi bare vil vise et gitt antall tegn.
Denne har heldigvis ingen dependencies da, så det ville ikke balle veldig på seg i node_modules

Reply:
jeg sjekket at den ikke hadde noen dependencies før jeg tok den i bruk. vi kunne lagd funksjonen selv i utils, men ser ikke helt poenget i det når det fungerer out of the box og ikke har noen dependencies

Reply:
støtter bruk av pakke her. slikt er lett å bruke alt for mye tid på.